src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -364,7 +364,7 @@ export default {
 					const key = contactId + '~' + addressbookId
 					const updated = this.$store.getters.getContact(key)
 					this.$store.dispatch('fetchFullContact', { contact: updated })
-					this.$store.dispatch('updateContact', updated)
+					this.$emit('updateLocalContact', updated)
```

Comment:
I have the impression that this doesn't call the updateLocalContact function. Where can I find information about the $emit ?

Reply:
https://vuejs.org/v2/guide/components-custom-events.html